package.json

```diff
@@ -78,11 +78,16 @@
     "eslint": "^7.19.0",
     "eslint-plugin-import": "^2.22.0",
     "eslint-plugin-prefer-arrow": "^1.2.3",
+    "execa": "^5.0.0",
+    "fixturify-project": "^4.0.2",
     "jest": "^27.0.0",
     "semantic-release": "^17.3.8",
     "ts-jest": "^27.0.0",
     "typescript": "^3.9.9"
   },
+  "resolutions": {
+    "resolve-package-path": "^4.0.3"
```

Comment:
Just out of curiosity, why is this necessary?

Reply:
There was a bug in version 3 of `resolve-package-path` that I [fixed](https://github.com/stefanpenner/resolve-package-path/pull/38). That fix has not made it into `fixturify-project` yet, but is largely safe since it's just a change to the files that were published. I'm opening a PR right now to roll that into `fixturify-project`.